examples/demo/src/react-admin.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-admin';
```

Comment:
what's that for?

Reply:
Because TS complain about react-admin not having types